### ISSUE TITLE
Insufficient differentiation for active work filter

### ISSUE THREAD

**Issue by dcruz** [minuscode/minuscode.github.io#38]
Active work filter doesn't have enough visual differentiation.
It used #932a3a instead of the default #c94055

For a perceptible differentiation we need both a change on color and form.
![work type filters](https://cloud.githubusercontent.com/assets/401064/15288751/1b5160c0-1b65-11e6-9b24-34c1e05058ae.png)
